README.md

```diff
@@ -1,36 +1,93 @@
+### Outline
+
+- [Overview](#overview)
+- [Docker Image Variants](#docker-image-variants)
+- [Additional Docker Examples](#additional-docker-examples)
+    - [How to use `theiaide/theia` image](#how-to-use-theiaidetheia-image)
+- [Electron Applications](#electron-apps)
+- [Running & Debugging](#running-&-debugging)
+- [Tips & Tricks](#tips-&-tricks)
+    - [Build Options](#build-options)
+- [Contributing](#contributing)
 - [License](#license)
 
-## Theia Docker
+<br />
+
+### Overview
+
+This repository contains a collection of Theia-based IDE applications (both electron desktop apps and docker images) as examples, for demo and continuous-integration purposes.
+
+### Docker Image Variants
+
+<img
+    src='./assets/docker.png'
+    alt='docker logo'
+    width="125px"
+/>
 
-[![dockeri.co](http://dockeri.co/image/theiaide/theia)](https://hub.docker.com/r/theiaide/theia/)
+| Image Name | Description | Documentation |
+|:---|:---|:---|
+| [theiaide/theia](https://hub.docker.com/r/theiaide/theia) | Theia-based JavaScript/TypeScript (Web Technologies) example application | |
 
-`theiaide/theia` image contains an example of Theia based IDE for Web Developers.
+_Other Variants_:
+
+| Image Name | Description | Documentation |
+|:---|:---|:---|
+| [theiaide/theia-cpp](https://hub.docker.com/r/theiaide/theia-cpp) | Theia-based C/C++ example application | [docs](./theia-cpp-docker/README.md) |
+| [theiaide/theia-dart](https://hub.docker.com/r/theiaide/theia-dart) | Theia-based Dart example application | |
+| [theiaide/theia-full](https://hub.docker.com/r/theiaide/theia-full) | Theia-based example application with support for multiple languages | |
+| [theiaide/theia-go](https://hub.docker.com/r/theiaide/theia-go) | Theia-based Go example application | [docs](./theia-go-docker/README.md) |
+| [theiaide/theia-java](https://hub.docker.com/r/theiaide/theia-java) | Theia-based Java example application | |
+| [theiaide/theia-python](https://hub.docker.com/r/theiaide/theia-python) | Theia-based Python example application | [docs](./theia-python-docker/README.md) |
+| [theiaide/theia-php](https://hub.docker.com/r/theiaide/theia-php) | Theia-based PHP example application | [docs](./theia-php-docker/README.md) |
+| [theiaide/theia-ruby](https://hub.docker.com/r/theiaide/theia-ruby) | Theia-based Ruby example application | |
+| [theiaide/theia-rust](https://hub.docker.com/r/theiaide/theia-rust) | Theia-based Rust example application | [docs](./theia-rust-docker/README.md) |
+| [theiaide/theia-swift](https://hub.docker.com/r/theiaide/theia-swift) | Theia-based Swift example application | |
+| [theiaide/yangster](https://hub.docker.com/r/theiaide/yangster) | Theia-based YANG example application | |
+
+---
+
+<br />
+
+### Additional Docker Examples
+
+| Image Name | Description | Documentation |
+|:---|:---|:---|
+| `theia-deb-build-docker` | Example on how to package the IDE into a Debian package | [docs](./theia-deb-build-docker/README.md) |
```

Comment:
Should there be a link to the image here ! 
ex: theia-deb-build-docker !

Reply:
The following is not published on dockerhub, it's only an example on how to package (as outline in the description).

Reply:
When I look at the preview pahe of the README, the title says: Additional Docker Examples"

It is like those are docker images as well

Reply:
We have a section of docker images, and docker examples.
The docker examples are simply examples on how to perform specific actions such as packaging for rpm and so on.